packages/react-components/react-shared-contexts/src/CustomStyleHooksContext/CustomStyleHooksContext.ts

```diff
@@ -119,6 +119,7 @@ export type CustomStyleHooksContextValue = Partial<{
   useMessageBarActionsStyles_unstable: CustomStyleHook;
   useMessageBarGroupStyles_unstable: CustomStyleHook;
   useToasterStyles_unstable: CustomStyleHook;
+  /** @deprecated TeachingPopover wrapper has no styles (non-UI hooks only). */
   useTeachingPopoverStyles_unstable: CustomStyleHook;
   useTeachingPopoverActionsStyles_unstable: CustomStyleHook;
   useTeachingPopoverBodyStyles_unstable: CustomStyleHook;
```

Comment:
_I can't comment on the image (`packages/react-components/react-teaching-popover-preview/etc/images/swapImage.png`)._

We should use images from CDN in examples, as I recall this works, https://github.com/microsoft/fluentui/blob/master/packages/fluentui/docs/README.md#faq

Reply:
Resolution tracked by #30127.